containers/self-hosted-runner/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Clone the repo.
+mkdir current-checkout
+git clone ${GITHUB_REPOSITORY} current-checkout
+cd current-checkout
+
+# Checkout the SHA we want to test.
+git fetch ${GITHUB_REF}
+git checkout ${GITHUB_SHA}
+
+# Test.
+cargo make test
```

Comment:
This will need to change to `cargo make build && cargo make integration` once @lkatalin's PR is merged: https://github.com/enarx/enarx/pull/591